modules/ROOT/pages/afm-mask-info-in-monitors.adoc

```diff
@@ -0,0 +1,55 @@
+= Using Secrets to Mask Sensitive Information in Monitors
+
+In monitors that you run in a private location, you can mask sensitive information such as headers, token values, username and password, and body values.
+
+To mask the header information:
+
+. <<configure-secret,Configure a shared secret>> in Secrets Manager for the information you want to mask. 
+. <<add-secret-to-monitor,Modify your monitor>> to use the information from Secrets Manager. 
+. <<grant-access-to-secret,Grant the monitor permission>> to access the secret, which creates an alias for the secret.
+. To deploy your changes, <<deploy-updated-monitor,upload the modified monitor>> to Functional Monitoring.
+
+// Procedure to add secrets in Secret Manager
+
+include::_partials/afm-task-config-secrets.adoc[tag=config-in-secrets-mgr,leveloffset=+1]
+
+[[add-secret-to-monitor]]
+== Add the Secret to Your Monitor
+
+Modify your monitor to use the information you configured in Secrets Manager.
+
+To add the secret to a monitor:
+
+. Create or download a monitor. See xref:afm-create-monitor.adoc[Creating Monitors] or xref:afm-download-test.adoc[Downloading Monitors].
+. Modify the `main.dwl` file to replace the sensitive information with the client ID and client secret you configured in Secrets Manager. In the following example, the sensitive information in headers are masked. The 'clientId-alias' and 'clientSecret-alias' are aliases for the secrets stored in Secrets Manager. See <<grant-access-to-secret>> to learn how to generate the alias.
```

Comment:
Suggestion: Put "In the following example" on a new line so it stands out more.

Reply:
Good call